coredb-operator/src/controller.rs

```diff
@@ -86,6 +86,25 @@ pub struct Context {
     pub metrics: Metrics,
 }
 
+#[derive(Clone, Debug, Deserialize, JsonSchema, Serialize, PartialEq)]
+pub struct Extension {
+    pub name: String,
+    pub version: String,
+    pub enabled: bool,
+    pub schema: String,
+}
+
+impl Default for Extension {
+    fn default() -> Self {
+        Extension {
+            name: "pg_stat_statements".to_owned(),
+            version: "1.9".to_owned(),
+            enabled: true,
+            schema: "postgres".to_owned(),
+        }
+    }
+}
```

Comment:
```suggestion
```
I think the default should be an empty vec.

Reply:
What's an example case for using this? I could see us using it as a default CRD value if we wanted this extension enabled whenever the user provides nothing in `spec.extensions` (like you had previously configured), but we want that to be empty in that case.

Reply:
I use `::Default()` a lot in tests or wherever I want to construct a default set of values for struct.

Reply:
:+1: seems reasonable!

Reply:
`default_extensions` is an empty vec regardless of implementing Default on the Extension